apps/condo/domains/miniapp/components/AppDescription/TopCard.tsx

```diff
@@ -0,0 +1,166 @@
+
+const TITLE_VERTICAL_MARK = 720
+
+const TopCardTitle: React.FC<TopCardTitleProps> = ({ logoSrc, title, description }) => {
+    const { width } = useWindowSize()
+    const isSmallLayout = Boolean(width && width < TITLE_VERTICAL_MARK)
+
+    const rowMargins: CSSProperties = isSmallLayout ? { marginTop: 180 } : { marginLeft: 200, marginTop: 12 }
+
+    return (
+        <>
+            <LogoContainer centered={isSmallLayout}>
+                <Image src={logoSrc || FALLBACK_IMAGE} style={IMAGE_STYLES} preview={false} fallback={FALLBACK_IMAGE}/>
+            </LogoContainer>
+            <Row gutter={[0, 16]} style={rowMargins}>
+                <Col span={24}>
+                    <Typography.Title level={4} style={WRAP_TEXT_STYLES}>
+                        {title}
+                    </Typography.Title>
+                </Col>
+                <Col span={24}>
+                    <Typography.Paragraph style={WRAP_TEXT_STYLES} type={'secondary'}>
+                        {description}
+                    </Typography.Paragraph>
+                </Col>
+            </Row>
+        </>
+    )
+}
+
+
+
+export const TopCard: React.FC<TopCardProps> = ({
+    developer,
+    published,
+    title,
+    description,
+    partnerUrl,
+    tag,
+}) => {
+    const intl = useIntl()
+    const DeveloperMessage = intl.formatMessage({ id: 'Developer' })
+    const PublishedMessage = intl.formatMessage({ id: 'Published' })
+    const PartnerSiteMessage = intl.formatMessage({ id: 'services.PartnerSite' })
+
+    const { width } = useWindowSize()
+    const isSmallLayout = Boolean(width && width < TITLE_VERTICAL_MARK)
+
+    const router = useRouter()
+
+    return (
+        <CardWrapper centered={isSmallLayout}>
+            <Card bordered={true} title={<TopCardTitle title={title} description={description} partnerUrl={partnerUrl} tag={tag}/>}>
+                <Space size={[40, 0]} direction={'horizontal'} wrap={true}>
+                    <Typography.Text ellipsis={true}>
+                        <Typography.Text type={'secondary'}>
+                            {DeveloperMessage}:&nbsp;
+                        </Typography.Text>
+                        <Typography.Text strong>
+                            {developer}
+                        </Typography.Text>
+                    </Typography.Text>
+                    <Typography.Text ellipsis={true}>
+                        <Typography.Text type={'secondary'}>
+                            {PublishedMessage}:&nbsp;
+                        </Typography.Text>
+                        <Typography.Text strong>
+                            {dayjs(published).format('DD.MM.YYYY')}
+                        </Typography.Text>
+                    </Typography.Text>
+                    {
+                        partnerUrl && (
+                            <Typography.Link onClick={() => router.push(partnerUrl)}>
```

Comment:
We're trying not to use anonymous functions inside render section of component. [Check this out ](https://user3141592.medium.com/react-gotchas-and-best-practices-2d47fd67dd22#:~:text=Because%20we%20are%20passing%20anonymous,method%20like%20onClick%3D%7Bthis.)

Reply:
Will fix it inside 2420 PR: https://github.com/open-condo-software/condo/pull/1277

Reply:
https://github.com/open-condo-software/condo/pull/1277/commits/b850023cf7417813d227e142be2d75bf46bcf292

Done in child PR